src/providers/RelayProvider.ts

```diff
@@ -101,6 +101,14 @@ export class RelayProvider implements TLProvider {
     return utils.formatToAmount(utils.calcRaw(balance, 18), 18)
   }
 
+  /**
+   * Returns the version of the currently configured relay server.
+   * @returns Version of relay in the format `relay/vX.X.X`.
```

Comment:
the idea was that it is `<name>/vX.X.X`, so for a different implementation is might say `myAwsomeRustRelay/v0.2.2`